src/main/java/com/woowacamp/soolsool/core/cart/dto/response/CartItemResponse.java

```diff
@@ -0,0 +1,29 @@
+package com.woowacamp.soolsool.core.cart.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.woowacamp.soolsool.core.cart.domain.CartItem;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class CartItemResponse {
+
+    private final Long id;
+    private final String liquorName;
+    private final String liquorPrice;
+    private final String imageUrl;
+    private final Integer liquorQuantity;
+
+    @JsonCreator
+    public static CartItemResponse from(@JsonProperty final CartItem cartItem) {
```

Comment:
```suggestion
    public static CartItemResponse from(final CartItem cartItem) {
```
`CartItemModifyRequest` 코멘트 참고해주세요~